src/msix/AppxPackageObject.cpp

```diff
@@ -331,9 +331,9 @@ namespace MSIX {
                 APPX_BUNDLE_PAYLOAD_PACKAGE_TYPE packageType;
                 ThrowHrIfFailed(package->GetPackageType(&packageType));
                 
+                auto bundlePackageInfo = package.As<IAppxBundleManifestPackageInfo>();

```

Comment:
>auto bundlePackageInfo = package.As<IAppxBundleManifestPackageInfo>(); [](start = 16, length = 70)

No need for this QI,  package is already an IAppxBundleManifestPackageInfo. you can rename it to make clear what it is